src/app/layout.tsx

```diff
@@ -12,7 +13,10 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body>{children}</body>
+      <body>
+        <Navbar />
```

Comment:
since we only want the navbar on the dashboard page, we'll want to move this component to `uw-211/src/app/dashboard/page.tsx`.

Reply:
Fixed in [e15b7b8](https://github.com/hack4impact-utk/uw-211/pull/32/commits/e15b7b8b9386269424bea8f4b95551552db09f4b)